mode/examples/Topics/Image Processing/Extrusion/Extrusion.pyde

```diff
@@ -25,6 +27,7 @@ def setup():
 
 def draw():
     global angle
+    global img
```

Comment:
This isn't necessary, as img is not being written to.

Indeed, the whole img variable isn't needed, since only the width and height are used here.